src/components/chip/chip.e2e.ts

```diff
@@ -25,6 +25,16 @@ describe("calcite-chip", () => {
     expect(eventSpy).toHaveReceivedEvent();
   });
 
+  it("should receive focus when clicked clicked", async () => {
```

Comment:
Can you also set up the `focusable` helper from `commonTests.ts`? Right now, it only takes care of calling `setFocus`, but I think we could enhance it to cover click and tabbing.